stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -658,10 +658,7 @@ pub enum TenureChangeError {
 
 /// Schnorr threshold signature using types from `wsts`
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct ThresholdSignature {
-    R: wsts::Point,
-    z: wsts::Scalar,
-}
+pub struct ThresholdSignature(pub wsts::common::Signature);
```

Comment:
I think we have two structs for this in-flight -- we have `ThresholdSignature` and `SchnorrSignature`.  Can we pick one? cc @jferrant

Reply:
I don't see `SchnorrSignature` in the codebase. Is it just a type alias for `common::Signature` or is it a wrapper like `ThresholdSignature`? I implemented it this way so I could `impl StacksMessageCodec` for it, but I have no preference for what we should name it.

Reply:
`SchnorrSignature` is in https://github.com/stacks-network/stacks-core/pull/4039.  Maybe this PR can be rebased on it?

Reply:
@jcnelson this PR is tiny by comparison with #4039 I think this should go in first then that can be rebased on this

Reply:
Very well

Reply:
I see... `SchnorrSignature` is implemented as a single-element tuple with a byte array, and `ThresholdSignature` is a single-element tuple with `common::Signature`. I think `ThresholdSignature` is a little simpler and allows for zero-cost, zero-risk conversions to and from `common::Signature'.

Probably best to ask @xoloki which implementation and naming convention we should go with, as there are all sorts of subtleties in cryptographic code that I don't fully understand

Reply:
Now that we have proper `serde` support I recommend the `ThresholdSignature` approach where we just wrap `common::Signature`.  When `serde` support was shakey I used to recommend byte arrays to make sure people did explicit conversions.